sandbox/settings.py

```diff
@@ -553,6 +553,18 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
     # Sentry
     SENTRY_DSN = values.Value(None, environ_name="SENTRY_DSN")
 
+    # Tracking configuration
+    WEB_ANALYTICS_GOOGLE_ANALYTICS_ID = (
```

Comment:
The case for calling it `WEB_ANALYTICS_ID` is that everything would work the same for developpers using another tool, like Xiti for example. They would just need to provide the Xiti ID and override the template... In the overriding template we put the Xiti js code and can still refer to the `WEB_ANALYTICS_ID` variable in the context. It's good if we can avoid to modify the context processor and setting names just to use another analytics tool.

Reply:
Ok.
I thought that it was possible to add a setting to a django template directly, but after some googling I found that it requires a context processor. So you have you're right!
If we have in the future another web analytics solution on the richie core, then we need to include an additional setting. If we include another setting `WEB_ANALYTICS_PROVIDER`, then we could make the `WEB_ANALYTICS_LOCATION` as optional. 
So if you agree we would use this settings:
* `WEB_ANALYTICS_ID` - the Google Analytics tracking id, Xiti id, or other web tracking id's
* `WEB_ANALYTICS_LOCATION` - with supported values of `head` (default) or `footer`, the Google Analytics recommends to put the js code on the head, but you have your Xiti js on (body) footer.
* `WEB_ANALYTICS_PROVIDER` - for now the richie core only supports `GOOGLE_ANALYTICS`.
The web analytics dimensions would be loaded if the setting `WEB_ANALYTICS_ID` is provided.

Reply:
Sorry I didn't get why the WEB_ANALYTICs_PROVIDER setting. 
The best test to see if analytics are activated is probably `if WEB_ANALYTICS_ID`?
All this is assuming that there can be only one analytics solution activated, not several at the same time. Do you think that it is common to have several analytics tool?

Reply:
I think is very unlikely that requirement. One or another.
But, in future, if we need to have support for more than one analytic software directly on Richie core.Then we need to know when to include the Google Analytics JS or other XPTO Analytics JS.

Because we have multiple if's on the code.
To add the dimensions on context processors, check if `WEB_ANALYTICS_ID` setting exists.
To add the Google Analytics JS the setting `WEB_ANALYTICS_PROVIDER` need to exist and have the value `GOOGLE_ANALYTICS`.

With this, it would be very easy to customize for Xiti on FUN installation:
* `WEB_ANALYTICS_ID` your Xiti tracking code
* `WEB_ANALYTICS_LOCATION` - with `footer` value
* `WEB_ANALYTICS_PROVIDER` - not defined
* override the `web_analytics.html`